<launch>
    <arg name="device_ip" default="192.168.1.200"/> <!--雷达ip-->
    <arg name="msop_port" default="2368"/>          <!--雷达目的数据端口-->
    <arg name="difop_port" default="2369"/>         <!--雷达目的设备端口-->
    <arg name="use_time_service" default="false"/>  <!--是否使用授时-->

    <node pkg="lslidar_driver" type="lslidar_driver_node" name="lslidar_driver_node" output="screen">
        <!--param name="pcap" value="$(find lslidar_driver)/pcap/LS500.pcap"/-->
        <param name="lidar_ip" value="$(arg device_ip)"/>
        <param name="msop_port" value="$(arg msop_port)"/>
        <param name="difop_port" value="$(arg difop_port)"/>
        <param name="add_multicast" value="false"/>
        <param name="group_ip" value="224.1.1.2"/>
        <param name="frame_id" value="laser_link"/> <!--点云坐标系id-->
        <param name="is_add_frame_" value="false"/>
        <param name="min_range" value="0.15"/>      <!--雷达扫描最小距离-->
        <param name="max_range" value="500"/>       <!--雷达扫描最大距离-->
        <param name="scan_start_angle" value="-60"/>    <!--雷达扫描最小角度-->
    	<param name="scan_end_angle" value="60"/>       <!--雷达扫描最大角度-->
        <param name="Vertical_angle_compensation" value="-28.0"/>   <!--垂直角度补偿-->
        <param name="pointcloud_topic" value="lslidar_point_cloud"/><!--点云话题名称-->
        <param name="use_time_service" value="$(arg use_time_service)"/>
    </node>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lslidar_driver)/rviz/lslidar_ls.rviz" output="screen"/>

</launch>
